src/libs/ReportActionsUtils.js

```diff
@@ -28,6 +28,30 @@ Onyx.connect({
     callback: val => isNetworkOffline = lodashGet(val, 'isOffline', false),
 });
 
+/**
+ * Sort an array of reportActions by their created timestamp,
+ * using actionName as a secondary sorting parameter,
+ * finally falling back on reportActionID in case multiple reportActions of the same actionName are created in the same millisecond.
+ *
+ * @param {Array} reportActions
+ * @param {Boolean} [shouldInvertSortingOrder]
+ * @returns {Array}
+ */
+function sortReportActions(reportActions, shouldInvertSortingOrder = false) {
+    if (!_.isArray(reportActions)) {
+        throw new Error(`ReportActionsUtils::sortReportActions requires an array, received ${typeof reportActions}`);
```

Comment:
```suggestion
        throw new Error(`ReportActionsUtils.sortReportActions requires an array, received ${typeof reportActions}`);
```
NAB, but I didn't see any other place where we use `::` like in PHP

Reply:
Updated